2020q2/kde.md

```diff
@@ -0,0 +1,50 @@
+## KDE on FreeBSD ##
+
+Link:    [KDE FreeBSD](https://freebsd.kde.org/)  
+Link:    [KDE Community FreeBSD](https://community.kde.org/FreeBSD)  
+
+Contact: Adriaan de Groot <kde@FreeBSD.org>  
+
+The *KDE on FreeBSD* project packages the software produced by
+the KDE Community for FreeBSD. The software includes a
+full desktop environment KDE Plasma, IDE [KDevelop](https://www.kdevelop.org/),
+a PIM suite [Kontact](https://kontact.kde.org/)
+and hundreds of other applications that can be used on
+any FreeBSD desktop machine.
+
+This quarter has been an ever-so-peculiar one. While we are used
+to working remotely, collaborating over the internet to update the ports
+tree, it's *qualitatively* different when
+the whole world locks down. Software continues to be released,
```

Comment:
s/Software/Meanwhile, software/
The reason being, it stands out as being a regular occurrence. :)